include/umf/experimental/ctl.h

```diff
@@ -19,32 +19,29 @@ extern "C" {
 ///
 /// @brief Get value of a specified attribute at the given name.
 /// @param name name of an attribute to be retrieved
-/// @param ctx pointer to the pool or the provider
 /// @param arg [out] pointer to the variable where the value will be stored
 /// @param size size of the value, depends on the context
 /// @return UMF_RESULT_SUCCESS on success or UMF_RESULT_ERROR_UNKNOWN on failure.
 ///
-umf_result_t umfCtlGet(const char *name, void *ctx, void *arg, size_t size);
+umf_result_t umfCtlGet(const char *name, void *arg, size_t size, ...);
```

Comment:
add information that the last parameter is an optional list of va arguments defined by the query here and in other umfCtl* functions

Reply:
done